### ISSUE TITLE
[Organisation] Accessibility improvements

### ISSUE THREAD

**Issue by CanadaHonk** [AlternativeFFFF/Alt-F4#83]
Changes known / to be done:
- [x] Alternate tags for post thumbnail images on the index page
  - [X] Issue made: #81
  - [X] PR made: #82
  - [x] PR merged

- [X] HTML tag should have `lang` attribute featuring the language code for the current page
  - [X] Issue made: #84
  - [X] PR made: #85
  - [X] PR merged

- [x] Use `em` / `strong` instead of `i` tags
  - [X] Issue made: #225
  - [X] PR made: #226
  - [x] PR merged

Changes which were thought to be an issue and no longer are
- ~~The `a` (link) element for clicking on the thumbnail image taking to the post possibly interferes with screen readers~~
  - ~~Issue made: #120~~
  - ~~PR made~~
  - ~~PR merged~~

(More will be added as discovered.)

**Comment by CanadaHonk** [comment]
Closing - just using specific issues and tracking them individually instead.
